processor/src/range/request.rs

```diff
@@ -84,9 +92,13 @@ impl CycleRangeChecks {
 impl LookupTableRow for CycleRangeChecks {
     /// Reduces this row to a single field element in the field specified by E. This requires
     /// at least 1 alpha value. Includes all values included at this cycle from all processors.
-    fn to_value<E: FieldElement<BaseField = Felt>>(&self, alphas: &[E]) -> E {
-        let stack_value = self.to_stack_value(alphas);
-        let mem_value = self.to_mem_value(alphas);
+    fn to_value<E: FieldElement<BaseField = Felt>>(
+        &self,
+        main_trace: &Matrix<Felt>,
+        alphas: &[E],
+    ) -> E {
+        let stack_value = self.to_stack_value(alphas, main_trace);
+        let mem_value = self.to_mem_value(alphas, main_trace);
```

Comment:
I would flip the order of the inputs to `to_stack_value` and `to_mem_value` as well, to be consistent with the rest